src/components/AppHeader.vue

```diff
@@ -121,4 +130,22 @@ const toggleClass = (e:Event) => {
     }
     (e.target as HTMLElement).classList.add('text-primary-600','pointer-events-none')
 }
+const changeTableName = async (e:Event) =>{
+    (e.target as HTMLElement).classList.add('hidden');
+    const el = (e.target as HTMLElement)!.parentElement!.lastElementChild;
+    el!.classList.remove('hidden');
+    await nextTick();
+    (el as HTMLInputElement).focus();
+}
+
+const submitChange = (e:Event) => {
+    const el = e.target as HTMLInputElement;
+    el.value.trim() === "" ? (e.target as HTMLInputElement).value = "Untitled" : 1
+    el.dispatchEvent(new Event('input'));
+    el.classList.add('hidden');
+    el.parentElement!.querySelector('#table_name-span')!.classList.remove('hidden')
```

Comment:
This is doesn't look right so let's talk (I will schedule a meeting some time tomorrow)